main.nf

```diff
@@ -623,7 +625,7 @@ process dupradar {
     script: // This script is bundled with the pipeline, in NGI-RNAseq/bin/
     def paired = single ? 'FALSE' :  'TRUE'
     """
-    dupRadar.r $bam_md $gtf $paired
+    dupRadar.r $bam_md $gtf $paired ${params.rlocation}
```

Comment:
You need the line above where I defined that: `def rlocation = params.rlocation ?: ''`